src/nni_manager/rest_server/restValidationSchemas.ts

```diff
@@ -31,10 +31,12 @@ export namespace ValidationSchemas {
                 passwd: joi.string(),
                 sshKeyPath: joi.string(),
                 passphrase: joi.string(),
-                gpuIndices: joi.string()
+                gpuIndices: joi.string(),
+                maxTrialNumOnEachGPU: joi.number()
```

Comment:
suggest change config name to: maxTrialNumPerGpu

Reply:
changed to use maxTrialNumPerGpu.